assume/scenario/loader_csv.py

```diff
@@ -815,7 +794,7 @@ def run_learning(
         if (
             episode % validation_interval == 0
             and episode
-            > world.learning_role.episodes_collecting_initial_experience
+            >= world.learning_role.episodes_collecting_initial_experience
```

Comment:
as the first episode is 1, you probably want to have > here - otherwise you would get no run without `initial_experience` when `episodes_collecting_initial_experience=1`

Reply:
since it is initial_experience+validation_interval, it will be at least 2, so this should be fine. Otherwise with > the first evaluation is done very late. For example, if eval=5 and init_experience=5, the first evaluation happens after episode=15